packages/lodestar/src/executionEngine/http.ts

```diff
@@ -68,24 +69,77 @@ export class ExecutionEngineHttp implements IExecutionEngine {
   async executePayload(executionPayload: merge.ExecutionPayload): Promise<ExecutePayloadResponse> {
     const method = "engine_executePayloadV1";
     const serializedExecutionPayload = serializeExecutionPayload(executionPayload);
-    const {status, latestValidHash} = await this.rpc.fetch<
-      EngineApiRpcReturnTypes[typeof method],
-      EngineApiRpcParamTypes[typeof method]
-    >({
-      method,
-      params: [serializedExecutionPayload],
-    });
+    const {status, latestValidHash, validationError} = await this.rpc
+      .fetch<EngineApiRpcReturnTypes[typeof method], EngineApiRpcParamTypes[typeof method]>({
+        method,
+        params: [serializedExecutionPayload],
+      })
+      /**
+       * If there are errors by EL like connection refused, internal error, they need to be
+       * treated seperate from being INVALID. For now, just pass the error upstream.
+       */
+      .catch((e: Error) => {
+        let errResponse: EngineApiRpcReturnTypes[typeof method];
+        if (e instanceof TimeoutError) {
+          errResponse = {status: ExecutePayloadStatus.UNAVAILABLE, latestValidHash: null, validationError: e.message};
+        } else if (e instanceof ErrorAborted) {
+          errResponse = {status: ExecutePayloadStatus.UNAVAILABLE, latestValidHash: null, validationError: e.message};
+
+          // What would be better way of handling this
+        } else if (e.message === "Only absolute URLs are supported") {
+          errResponse = {status: ExecutePayloadStatus.UNAVAILABLE, latestValidHash: null, validationError: e.message};
```

Comment:
can we merge these e conditions into 1? `if (condition1 || condition2 || condition3)`